runatlantis.io/docs/repo-level-atlantis-yaml.md

```diff
@@ -46,11 +46,13 @@ need to be defined.
 ```yaml
 version: 3
 automerge: true
+delete_source_branch_on_merge: true
```

Comment:
As automerge is missing in the server side yaml (only --automerge flag can be applied during startup) so there is no way to aggregate autoconfig and delete_source_branch_on_merge in server side config yaml. Therefore aggregating automerge and delete_source_branch_on_merge to one single block in repo config would cause two different "syntaxes", one in server.yaml which only contains delete_source_branch_on_merge and one in repo.yaml which would contain the agregated block.  I suggest to leave as it's now, and in a later phase the granurality of automerge can be increased (adding it to server.yaml) and at that point automerge and delete_source_branch can be aggregated into one single config block in both server.yaml and repo.yaml.

Reply:
Additionally as the project config does not contain automerge it cannot be aggregated with delete_source_branch_on_merge either on project level. I set delete_source_branch_on_merge to be optional with false default value on each level (server/repo/project) so it would not cause a breaking change, the currently running server instances would not require new config format.